packages/protocol-kit/src/adapters/web3/Web3Adapter.ts

```diff
@@ -273,13 +273,13 @@ class Web3Adapter implements EthAdapter {
   }
 
   async signTypedData(
-    safeTransactionEIP712Args: SafeTransactionEIP712Args,
+    safeEIP712Args: SafeEIP712Args,
     methodVersion?: 'v3' | 'v4'
   ): Promise<string> {
     if (!this.#signerAddress) {
       throw new Error('This method requires a signer')
     }
-    const typedData = generateTypedData(safeTransactionEIP712Args)
+    const typedData = generateTypedData(safeEIP712Args)
     let method = 'eth_signTypedData_v3'
     if (methodVersion === 'v4') {
       method = 'eth_signTypedData_v4'
```

Comment:
Wouldn't be good to use the SigningMethod enum here too?

There are many places in the code where these strings are written directly. Now you created the enum maybe you should use it in all those places too

Reply:
yes totally, i created it at a later stage and didn't make the susbstitutions